cmd/cli/install.go

```diff
@@ -189,6 +191,7 @@ func newInstallCmd(config *helm.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVar(&inst.deployJaeger, "deploy-jaeger", defaultDeployJaeger, "Deploy Jaeger in the namespace of the OSM controller")
 	f.StringVar(&inst.envoyLogLevel, "envoy-log-level", defaultEnvoyLogLevel, "Envoy log level is used to specify the level of logs collected from envoy and needs to be one of these (trace, debug, info, warning, warn, error, critical, off)")
 	f.BoolVar(&inst.enforceSingleMesh, "enforce-single-mesh", defaultEnforceSingleMesh, "Enforce only deploying one mesh in the cluster")
+	f.BoolVar(&inst.enablePrivilegedInitContainer, "enable-privileged-init-container", defaultEnablePrivilegedInitContainer, "Deploy privileged init container for pods in mesh")
```

Comment:
We should simply be using the `--set` option to override chart values going forward to avoid the need to wrap every chart value using a cli option.